otp/imx6ull-otp/imx6ull-otp.c

```diff
@@ -549,6 +557,48 @@ static int write_sn(const char *sn, int raw)
 }
 
 
+static int read_hwrev(int silent)
+{
+	uint32_t plc1 = *(otp.base + ocotp_gp2);
+
+	// NOTE: keeping (hw_rev - 1) in FUSE bits
+	uint8_t hw_rev = (plc1 >> 24 & 0xff) + 1;
+	if (silent == 0) {
+		printf("%u\n", hw_rev);
+	}
+
+	return hw_rev;
+}
+
+
+static int write_hwrev(uint8_t hw_rev)
+{
+	int ret = 0;
+
+	if ((read_hwrev(1) != 1) && !(common.force == 0)) {
```

Comment:
I don't know, maybe it's beneficial to use `!`? Just a suggestion
```suggestion
	if ((read_hwrev(1) != 1) && (common.force != 0)) {
```

Reply:
^C^V from other procedures, turns out this was always invalid in the code, `!` should not be there (lol)